### PR TITLE
Wk 169 refactor names

### PR DESCRIPTION
**Pull request by aditudorache**
This PR does some refactoring in variables and file names for a better readablility. Also minor style improvements are applyed

### CODE REVIEW

File: src/components/locationForm/components/RadioGroupInput.jsx [hunk 1]
```diff
@@ -4,12 +4,21 @@ import styled from '@datapunt/asc-core';
 
 const RadioLabelStyle = styled(Label)`
   font-weight: normal;
+
+  & > :first-child {
+    margin: 0;
+    padding-top: 5px;
```
Comment (janjaap) [inline src/components/locationForm/components/RadioGroupInput.jsx]: `themeSpacing()`?

File: src/components/locationForm/services/normalize.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    },
+  } = item;
+  return {
+    ...initalValues,
+    naam: description,
+    nummer: locatie_id,
+    coordinaten: `${coordinates[1]}, ${coordinates[0]}`,
+    spot_type,
+    jaar_blackspotlijst,
+    status: status,
```
Comment (janjaap) [inline src/components/locationForm/services/normalize.js]: or `status,`

File: src/components/locationForm/services/normalize.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    actiehouder: actiehouders,
+    taken: tasks,
+    start_uitvoering: getDate(start_uitvoering),
+    eind_uitvoering: getDate(eind_uitvoering),
+    jaar_oplevering: jaar_oplevering,
```
Comment (janjaap) [inline src/components/locationForm/services/normalize.js]: Or `jaar_oplevering,`

File: src/components/locationForm/services/normalize.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  } = item;
+  return {
+    ...initalValues,
+    naam: description,
+    nummer: locatie_id,
+    coordinaten: `${coordinates[1]}, ${coordinates[0]}`,
+    spot_type,
+    jaar_blackspotlijst,
+    status: status,
+    actiehouder: actiehouders,
```
Comment (janjaap) [inline src/components/locationForm/services/normalize.js]: Or `actiehouder,`

File: src/views/location/LocationPage.test.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import LocationPage from './LocationPage';
+
+describe('LocationPage', () => {
+  it('should render without errors', () => {
+    shallow(<LocationPage match={{ params: {} }} />);
```
Comment (janjaap) [inline src/views/location/LocationPage.test.js]: This test doesn't verify anything

File: src/components/locationForm/components/RadioGroupInput.jsx (hunk 1 above)
Comment (aditudorache) [reply src/components/locationForm/components/RadioGroupInput.jsx]: not appropriate here, is not aligning with other elements but with the native radio control. Will be replaced later by the styled radio button from `ASC`

File: src/components/locationForm/services/normalize.js (hunk 4 above)
Comment (aditudorache) [reply src/components/locationForm/services/normalize.js]: not possible, is plural

File: src/components/locationForm/services/normalize.js (hunk 3 above)
Comment (aditudorache) [reply src/components/locationForm/services/normalize.js]: thanks

File: src/components/locationForm/services/normalize.js (hunk 2 above)
Comment (aditudorache) [reply src/components/locationForm/services/normalize.js]: thanks

File: src/views/location/LocationPage.test.js (hunk 5 above)
Comment (aditudorache) [reply src/views/location/LocationPage.test.js]: unittests will follow

**Pull request merged**